db/seed_helper.rb

```diff
@@ -5,4 +5,15 @@ def create_roles_and_permissions
   Role.create!(:name => Role::DATA_OWNER_ROLE)
 end
 
-
+# Populates the languages table with the language names and codes from the languages CSV file
+def populate_languages
+  Language.delete_all
+  languages = {}
```

Comment:
No point having an empty hash, then copying what you want out of the CSV then iterating across it. Just construct Models as you are iterating across the CSV.